config/qualifications.yml

```diff
@@ -1,21 +1,31 @@
-# A fully-trained programmer should
 ---
-title: 'Have demonstrated understanding of Ruby core methods / Rails object extensions'
+title: 'Configure your laptop for development'
 description: >
+  go to https://strap.umasstransit.it/ and follow the instructions.
 subtasks:
-  - 'Ruby object types'
-  - 'Methods and functionality brought by Rails (try, 1.week, etc.)'
 ---
-title: 'Have demonstrated familiarity with MVC architecture and the Rails asset pipeline'
+title: 'Join UMTS on Github'
 description: >
+  Ask one of your supervisors or team leaders to get you added to
+  the relevant groups in our Github organization.
 subtasks:
-  - 'When to place logic in controller / model'
-  - 'What not to put in a view'
-  - 'Partials'
-  - 'Working securely with params (guarding at the borders)'
 ---
-title: 'Routing, especially member vs. collection routes'
+title: 'Read through documents related to university owned computers'
 description: >
+  Since you will be trusted with university-owned computers and potentially
+  sensitive data, you should review these documents
 subtasks:
-  - 'Default resource CRUD routes'
-  - 'Difference between GET and POST / links and buttons'
+  - '[Security Checklist](https://www.umass.edu/it/support/security/security-checklist-university-owned-computers)'
+  - '[Understanding Sensitive Data](https://www.umass.edu/it/support/security/understand-sensitive-data-umass-amherst)'
+  - '[Data Classification]9https://www.umass.edu/it/support/security/data-classification-umass-amherst)'
```

Comment:
`s/9/(/`

Reply:
What does that mean? Did I get "making links" wrong?

Reply:
Yeah, you missed the shift key. That `9` should be a parenthesis.

Reply:
O those are vim commands I am a dum